cram_3d_world/cram_bullet_reasoning/src/objects.lisp

```diff
@@ -358,16 +358,56 @@
       (loop for body in (rigid-bodies object)
             collecting (make-collision-information
                         :rigid-body-name (name body)
-                        :flags (collision-flags body))
-            do (setf (collision-flags body) :cf-static-object))))
+                        :flags (or (when (not (attached-objects object))
+                                     (collision-flags body))
+                                   (if (object-static-in-past-p object)
+                                       '(:cf-static-object)
+                                       NIL)))
```

Comment:
It should not be NIL, it should be the old collision information that is stored. The collision information should simply be taken over if the status of "attached" hasn't changed for the object. So, if it is already attached, perhaps nothing should happen at all. And the same during resetting, if it is still attached, nothing should happen (if that doesn't work, the stored collision information should be taken over).